src/extension/TeamCityEventListener.cs

```diff
@@ -37,7 +37,7 @@ namespace NUnit.Engine.Listeners
     // purely documentary since engines prior to 3.4 do not
     // check the EngineVersion property and will try to
     // load this extension anyway.
-    [Extension(Enabled = false, EngineVersion = "3.4")]
+    [Extension(Enabled = false, EngineVersion = "3.5")]
```

Comment:
This means that the extension will be ignored on NUnit 3.4.1. Is that what you really want? Since it's not a pre-release, users will automatically get this extension version and if they are on 3.4.1, it will refuse to work.